lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/reuse-existing-identity.yaml

```diff
@@ -266,6 +296,22 @@ states:
       end:
         targetState: UPDATE_DETAILS_PAGE
 
+  UPDATE_NAME_DOB_ALLOW_ACCOUNT_DELETION:
+    response:
+      type: page
+      pageId: update-name-date-birth
+      context: reuse
+    events:
+      continue:
+        targetState: RETURN_TO_RP
+      end:
+        targetState: DELETE_IDENTITY_PAGE
+
+  DELETE_IDENTITY_PAGE:
+    response:
+      type: page
+      pageId: delete-handover
```

Comment:
I think we might need a back event here:
    ```
events:
      back:
        targetState: UPDATE_NAME_DOB_ALLOW_ACCOUNT_DELETION
```

Reply:
Can you try this without - have a look at `ProcessJourneyEventHandler#BACK_EVENT`

If we're lucky, it should 'just work' 🤞

Reply:
yep, it just works!

Reply:
Oh nice - will take it out of mine too :)